src/main/cpp/geo/Profile.h

```diff
@@ -14,15 +14,27 @@ using ProfileRef = std::shared_ptr<const Profile>;
 class Profile
 {
 public:
-	Profile(std::vector<City>& cities) : cities(cities) {}
+	Profile(const std::vector<City>& cities, const GeoRectangle& simulation_area)
+	    : m_cities(cities), m_simulation_area(simulation_area)
+	{
+		std::sort(m_cities.begin(), m_cities.end(), [](const City& a, const City& b) {
+			return a.relative_population > b.relative_population;
+		});
+	}
 
 	/// Return a list of cities in this geodistribution profile.
-	const std::vector<City>& GetCities() const { return cities; }
+	/// The result is sorted in descending order of relative population.
+	const std::vector<City>& GetCities() const { return m_cities; }
+
+	/// Return the simulation area, which is defined as the the latitude/longitude
+	/// bounding geodesic rectangle of all the cities.
+	const GeoRectangle GetSimulationArea() const { return m_simulation_area; }
```

Comment:
Returning a `const` value doesn't actually mean anything. Please delete the leading `const` or change the return type to a reference.